notebooks/community/model_evaluation/automl_tabular_classification_model_evaluation.ipynb

```diff
@@ -0,0 +1,1433 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    @kfp.dsl.pipeline(name="vertex-evaluation-automl-tabular-feature-attribution-pipeline")
</pre><p>Prefer to switch the name to "vertex-evaluation-automl-tabular-classification-feature-attribution", removing "pipeline"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/911/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='59'/>

Reply:
<p>updated.</p>